test/augmentation/test_augmentation.py

```diff
@@ -0,0 +1,113 @@
+                                 [0., 0., 0.],
+                                 [1., 1., 0.]])  # 3 x 3
+
+        expected_transform = torch.tensor([[-1., 0., 3.],
+                                           [0., 1., 0.],
+                                           [0., 0., 0.]])  # 3 x 3
+
+        identity = torch.tensor([[1., 0., 0.],
+                                 [0., 1., 0.],
+                                 [0., 0., 1.]])  # 3 x 3
+
+        assert (f(input)[0] == expected).all()
+        assert (f(input)[1] == expected_transform).all()
+        assert (f1(input)[0] == input).all()
+        assert (f1(input)[1] == identity).all()
+
+    def test_batch_random_hflip(self):
+
+        f = kornia.RandomHorizontalFlip(p=1.0)
+        f1 = kornia.RandomHorizontalFlip(p=0.0)
+
+        input = torch.tensor([[[0., 0., 0.],
+                               [0., 0., 0.],
+                               [0., 1., 1.]]])  # 1 x 3 x 3
+
+        expected = torch.tensor([[[0., 0., 0.],
+                                  [0., 0., 0.],
+                                  [1., 1., 0.]]])  # 1 x 3 x 3
+
+        expected_transform = torch.tensor([[[-1., 0., 3.],
+                                            [0., 1., 0.],
+                                            [0., 0., 0.]]])  # 1 x 3 x 3
+
+        identity = torch.tensor([[[1., 0., 0.],
+                                  [0., 1., 0.],
+                                  [0., 0., 1.]]])  # 1 x 3 x 3
+
+        input = input.repeat(2, 1, 1)  # 2 x 3 x 3
+        expected = expected.repeat(2, 1, 1)  # 2 x 3 x 3
+        expected_transform = expected_transform.repeat(2, 1, 1)  # 2 x 3 x 3
+        identity = identity.repeat(2, 1, 1)  # 2 x 3 x 3
+
+        assert (f(input)[0] == expected).all()
+        assert (f(input)[1] == expected_transform).all()
+        assert (f1(input)[0] == input).all()
+        assert (f1(input)[1] == identity).all()
+
+    @pytest.mark.skip(reason="turn off all jit for a while")
+    def test_jit(self):
+        @torch.jit.script
+        def op_script(data: torch.Tensor) -> torch.Tensor:
+
+            return kornia.random_hflip(data)
+
+        input = torch.tensor([[0., 0., 0.],
+                              [0., 0., 0.],
+                              [0., 1., 1.]])  # 3 x 3
+
+        # Build jit trace
+        op_trace = torch.jit.trace(op_script, (input, ))
+
+        # Create new inputs
+        input = torch.tensor([[0., 0., 0.],
+                              [5., 5., 0.],
+                              [0., 0., 0.]])  # 3 x 3
+
+        input = input.repeat(2, 1, 1)  # 2 x 3 x 3
+
+        expected = torch.tensor([[[0., 0., 0.],
+                                  [0., 5., 5.],
+                                  [0., 0., 0.]]])  # 3 x 3
+
+        expected = expected.repeat(2, 1, 1)
+
+        actual = op_trace(input)
+
+        assert_allclose(actual, expected)
+
+    def test_gradcheck(self):
+
+        input = torch.tensor([[0., 0., 0.],
```

Comment:
Wouldn`t it better to use random for gradcheck, as we do typically?
E.g. https://github.com/kornia/kornia/blob/master/test/geometry/test_linalg.py#L76